src/Downloader/Args.cs

```diff
@@ -0,0 +1,156 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+public static class Args
+{
+    public static void ShowUsage(string? message = null)
+    {
+        Console.WriteLine($"Invalid or missing arguments.{(message is null ? "" : " " + message)}");
+        Console.WriteLine("  --token {github_token}");
+        Console.WriteLine("  --repo {org/repo1}[,{org/repo2},...]");
+        Console.WriteLine("  --label-prefix {label-prefix}");
+        Console.WriteLine("  [--issue-data {path/to/issues.tsv}]");
+        Console.WriteLine("  [--issue-limit {rows}]");
+        Console.WriteLine("  [--pull-data {path/to/pulls.tsv}]");
+        Console.WriteLine("  [--pull-limit {rows}]");
+        Console.WriteLine("  [--page-size {size}]");
+        Console.WriteLine("  [--page-limit {pages}]");
+        Console.WriteLine("  [--retries {comma-separated-retries-in-seconds}]");
+        Console.WriteLine("  [--verbose]");
+
+        Environment.Exit(1);
+    }
+
+    public static (
+        string org,
+        string[] repos,
+        string githubToken,
+        string? issuesPath,
+        int? issueLimit,
+        string? pullsPath,
+        int? pullLimit,
+        int? pageSize,
+        int? pageLimit,
+        int[] retries,
+        Predicate<string> labelPredicate,
+        bool verbose
```

Comment:
Is it worth converting this into a struct or a record? To simplify the code and make it more extensibility-friendly (e.g., adding a new property will be a simple task vs creating new variables all over the place).

Reply:
Yeah, that'd be all right. I was trying to keep it as simple as possible, but your illustration of moving to structs looks good.